src/py/flwr/simulation/ray_transport/ray_client_proxy_test.py

```diff
@@ -64,9 +64,13 @@ def get_properties(self, config: Config) -> Dict[str, Scalar]:
         return {"result": result}
 
 
-def get_dummy_client(cid: str) -> Client:
+def get_dummy_client(
+    node_id: int, partition_id: Optional[int]  # pylint: disable=unused-argument
+) -> Client:
     """Return a DummyClient converted to Client type."""
-    return DummyClient(cid).to_client()
+    if partition_id is None:
+        raise ValueError("`partition_id` is not set.")
+    return DummyClient(partition_id).to_client()
```

Comment:
Should we start to pass `node_id` instead of `partition_id` to `Client`/`NumPyClient` objects?

Reply:
This is just a test, but I could amend it to use `node_id`. 

Do you mean injecting `node_id` to all `Client/NumPyClient` ? How does doing so via the `Context`, which is already being injected into the clients (see https://github.com/adap/flower/blob/5fe288aec10386a9f479750d5f968f47236dece6/src/py/flwr/client/message_handler/message_handler.py#L108)

Reply:
My comment was more about consistency. `node_id` and `cid` refer to the same overall concept, whereas `partition_id` does not.

It would therefore be more consistent to use
```suggestion
    return DummyClient(cid=str(node_id)).to_client()
```
if a `cid: str` is expected.

Reply:
I'll make the change in #3699 once this PR is merged. I think it's easier that way